implementation/configuration/src/configuration_impl.cpp

```diff
@@ -333,7 +333,7 @@ bool configuration_impl::load(const std::string &_name) {
 
     // Tell, if reading of configuration file(s) failed.
     // (This may file if the logger configuration is incomplete/missing).
-    for (auto f : its_failed)
+    for (auto const& f : its_failed)
```

Comment:
Prefer `const auto` over `auto const`.

```suggestion
    for (const auto& f : its_failed)
```